firestore.rules

```diff
@@ -153,6 +157,10 @@ service cloud.firestore {
       allow write: if false;
     }
 
+    match /generatedSpeakers/{session}/feedback/{userId} {
```

Comment:
We can't assume that `generatedSpeakers` collection and subcollections won't randomly get deleted and regenerated so nothing should get written to them.